Jenkinsfile

```diff
@@ -71,14 +71,14 @@ pipeline {
 				// Convert the JaCoCo coverate to the Cobertura XML file format.
 				// This is done since the Jenkins JaCoCo plugin does not work well.
 				// See also JENKINS-212 on jira.catrob.at
-				sh "if [ -f '$JACOCO_UNIT_XML' ]; then ./buildScripts/cover2cover.py $JACOCO_UNIT_XML > $JAVA_SRC/coverage1.xml; fi"
+				sh "./buildScripts/cover2cover.py $JACOCO_UNIT_XML $JAVA_SRC/coverage1.xml"
```

Comment:
Should we guard those variables for whitespaces?
```
sh "./buildScripts/cover2cover.py \"$JACOCO_UNIT_XML\" \"$JAVA_SRC/coverage\"
```

Reply:
I don't think that this is necessary as white spaces in directories is not best-practice.
It is very unlikely that we will encounter white spaces, especially on Jenkins.